org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/utils/MarkdownConverterTest.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+* Copyright (c) 2019 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+
+package org.eclipse.lsp4xml.utils;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+/**
+ * MarkdownConverterTest
+ */
+public class MarkdownConverterTest {
```

Comment:
Could you write more tests with HTML that you wish to convert because i would like to investigate to use xmlscanner to generate markdown. Thanks!

Reply:
From what I have seen for XSD's, this seems to cover all the cases I could find.

Reply:
see https://github.com/angelozerr/lsp4xml/issues/245#issuecomment-457214267 and https://github.com/felixfbecker/PowerShellXSD/blob/master/Test.Types.ps1xml